skills/strategy-marketplace/SKILL.md

```
@@ -0,0 +1,219 @@
+import { purchaseStrategy } from "@aircade/strategy-marketplace";
+
+const access = await purchaseStrategy({
+  strategyId: "0x...",
+  buyerAgentId: "0x..."
+  // Payment handled via x402
+});
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)